<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileManager</class>
 <widget class="QWidget" name="FileManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>641</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Asset Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons.qrc">
    <normaloff>:/editor/icons/icons/assets.png</normaloff>:/editor/icons/icons/assets.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLineEdit" name="ln_search"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QTableView" name="tableView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>39</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>17</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
    </widget>
   </item>
   <item row="0" column="3" rowspan="6" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="FileViewer" name="fileViewer" native="true"/>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="grp_Mapper">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Caract√©ristiques</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="1">
         <widget class="QLineEdit" name="ln_Path">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QLineEdit" name="ln_UID">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QToolButton" name="btn_Path">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QLineEdit" name="ln_Name"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Chemin</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>UID</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Nom</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="bnt_Add">
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="btn_Remove">
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCommandLinkButton" name="btn_save">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Sauvegarder</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Recherche</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileViewer</class>
   <extends>QWidget</extends>
   <header>FileManager/FileViewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>bnt_Add</sender>
   <signal>clicked()</signal>
   <receiver>FileManager</receiver>
   <slot>createFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>871</x>
     <y>567</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Path</sender>
   <signal>clicked()</signal>
   <receiver>FileManager</receiver>
   <slot>browseForFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>853</x>
     <y>528</y>
    </hint>
    <hint type="destinationlabel">
     <x>610</x>
     <y>553</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>FileManager</receiver>
   <slot>rowChanged(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>698</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Remove</sender>
   <signal>clicked()</signal>
   <receiver>FileManager</receiver>
   <slot>removeFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>875</x>
     <y>641</y>
    </hint>
    <hint type="destinationlabel">
     <x>885</x>
     <y>651</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_save</sender>
   <signal>clicked()</signal>
   <receiver>FileManager</receiver>
   <slot>saveToFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>626</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>609</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ln_search</sender>
   <signal>textEdited(QString)</signal>
   <receiver>FileManager</receiver>
   <slot>search(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>461</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>rowChanged(QModelIndex)</slot>
  <slot>createFile()</slot>
  <slot>browseForFile()</slot>
  <slot>saveChanges()</slot>
  <slot>removeFile()</slot>
  <slot>playSound()</slot>
  <slot>saveToFile()</slot>
  <slot>search(QString)</slot>
 </slots>
</ui>
